src/main/java/com/thebois/models/beings/Colony.java

```diff
@@ -0,0 +1,45 @@
+package com.thebois.models.beings;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import com.thebois.models.Position;
+
+/**
+ * A Colony is a collection of Pawns that can be controlled by the player.
+ */
+public class Colony implements IBeingGroup {
+
+    private final Collection<IBeing> pawns = new ArrayList<>();
+
+    @Override
+    public Collection<IBeing> getBeings() {
+        return pawns;
+    }
+
+    @Override
+    public void createBeing() {
+        final Position spawnPosition = new Position();
+        pawns.add(new Pawn(spawnPosition));
+    }
```

Comment:
Might be wrong don't think we should have this method because we have no way of knowing whether or not, the being will spawn on top of something that it should not be able to stand on.